examples/http_log.rb

```diff
@@ -37,21 +38,13 @@ def response(env)
 
     process_time = Time.now.to_f - start_time
 
-    response_headers = {}
-    resp.response_header.each_pair do |k, v|
-      response_headers[to_http_header(k)] = v
-    end
+    response_headers = resp.response_header.raw
```

Comment:
in my environment, `response_header.raw()` returns well-formed hash like this:
```ruby
resp.response_header:
{"CONTENT_TYPE"=>"text/html",
 "DATE"=>"Fri, 05 Jun 2015 01:01:07 GMT",
 "CONNECTION"=>"close",
 "TRANSFER_ENCODING"=>"chunked"}
resp.response_header.raw:
{"Content-Type"=>"text/html",
 "Date"=>"Fri, 05 Jun 2015 01:01:07 GMT",
 "Connection"=>"close",
 "Transfer-Encoding"=>"chunked"}
```
( I used `pp resp.response_header` and `pp resp.response_header.raw`)